Hookies/Contexts/PreGameLobby/PreGameLobbyViewController.swift

```diff
@@ -165,24 +160,39 @@ class PreGameLobbyViewController: UIViewController {
         API.shared.gameplay.saveGameState(gameplay: gameplay)
     }
 
+    // swiftlint:disable line_length
+
     private func createPlayers(with lobby: Lobby) -> [Player] {
         var players: [Player] = []
         guard let currentId = API.shared.user.currentUser?.uid else {
             Logger.log.show(details: "current player not found", logType: .error)
             return players
         }
         let hostId = lobby.hostId
-        let hostCostume = lobby.costumesId[lobby.hostId] ?? CostumeType.getDefault()
-        guard let host = Player(playerId: hostId, playerType: .human, costumeType: hostCostume, isCurrentPlayer: currentId == hostId) else {
+        guard let hostCostume = lobby.costumesId[lobby.hostId] ?? CostumeType.getDefault() else {
+            return players
+        }
+        guard let host = Player(playerId: hostId, playerType: .human, costumeType: hostCostume, isCurrentPlayer: currentId == hostId, isHost: true) else {
             return players
         }
         players.append(host)
+
         for playerId in lobby.playersId.filter({ $0 != lobby.hostId }) {
-            let costume = lobby.costumesId[playerId] ?? CostumeType.getDefault()
-            guard let player = Player(playerId: playerId, playerType: .human, costumeType: costume, isCurrentPlayer: currentId == playerId) else {
+            guard let costume = lobby.costumesId[playerId] ?? CostumeType.getDefault() else {
```

Comment:
Wouldn't this always have a value that's not nil?

Reply:
If there are no cases in the enum then the getDefault method will return nil